csi.proto

```diff
@@ -1,6 +1,8 @@
 ////////
 syntax = "proto3";
-package csi;
+package csi.v1;
+
+option go_package = "csi";
```

Comment:
Hi @jdef,

Please consider adding the following after `option go_package`:

```proto
option java_package="csi";
option csharp_namespace="csi";
```

Reply:
As mentioned later in this thread, I think this only makes sense to do once we're generating bindings within the spec repo for these langs, otherwise it may be difficult to predict the full impact of changes on bindings generated from the spec.